### PR TITLE
Visible followers

### PR DESCRIPTION
**Pull request by JakobArnfred**
A users number of followers is now shown on the usertimeline. New separate issue is created for creating of tests for this feature

### CODE REVIEW

**Review by LavenderPlush — CHANGES_REQUESTED** [review]
This looks great!
If I understand correctly, this shows only the users own following?

A different solution could be changing the authorDTO to include follower count, so that an additional call the the database wasn't necessary

**Changes**:
It would be nice if the commented code in UserTimeline.cshtml.cs was removed if not used.

File: src/Chirp.Web/Pages/Public/UserTimeline.cshtml.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class UserTimelineModel : PageModel
 	private readonly IValidator<CreateCheepDTO> CheepValidator;
 	public required List<CheepDTO> Cheeps { get; set; }
 	public required List<AuthorDTO> Following { get; set; }
+	public int followerCount { get; set; }
```
Comment (Philipnah) [inline src/Chirp.Web/Pages/Public/UserTimeline.cshtml.cs:16]: This should follow the Pascal case naming convention.

**Review by Philipnah — CHANGES_REQUESTED** [review]
I agree that code that is commented out should be removed. Maybe a check for "if follower count = 1" then show "1 follower" instead of "1 followers"?

**Comment by Flakiator** [comment]
> This looks great! If I understand correctly, this shows only the users own following?

It also shows other users followers when you visit their profiles! 😎

**Review by LavenderPlush — APPROVED** [review]
These changes look great!

**Review by Philipnah — APPROVED** [review]

**Pull request merged**